Source/WebCore/css/CSSProperty.h

```diff
@@ -114,7 +114,17 @@ class CSSProperty {
     const StylePropertyMetadata& metadata() const { return m_metadata; }
     static bool isColorProperty(CSSPropertyID propertyId)
     {
-        return colorProperties.get(propertyId);
+        // colorProperties bitset does not include the first two CSSPropertyID (0 and 1),
+        // so we exclude them.
+
+        if (propertyId < firstCSSProperty)
+            return false;
+
+        unsigned index = propertyId - firstCSSProperty;
+        if (index >= numCSSProperties)
+            return false;
+
+        return colorProperties.get(index);
```

Comment:
How about defining numCSSPropertiesBitSetSize = numCSSProperties + firstCSSProperty instead and use it for `BitSet`?

Reply:
That could work too, yeah.

Reply:
Yeah, I think that's more efficient & simple :)